Update/some_omake.txt

```diff
@@ -1102,14 +1102,14 @@ void main()
 			NULL, " Anyway, let's get started.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s20/38/vtp_tamako1045", 256, TRUE);
 	OutputLine(NULL, "これから『ひぐらし』のアドバンスドストーリーを一緒に進めていく仲間としての最初の親睦ってことで、ね？」",
-			NULL, " This is our first opportunity to advance the story of \"Higurashi\" together, isn't it?\"", Line_Normal);
+			NULL, " This will be our first bonding sessions as the cast of these side stories \"Higurashi\", won't it?\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " This will be our first bonding session as the cast of the \"Higurashi\" Advanced Story series, won't it?\"", Line_Normal);
```